source/_components/switch.etherrain.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "EtherRain Switch"
+description: "How to turn on/off your irrigation valves"
+date: 2017-08-09 06:16
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: etherrain.jpg
+ha_category: Switch
+ha_release: 0.31
```

Comment:
HA release will most like need to be updated to 0.52  since 0.51 is rather imminent.